semantics/src/lib.rs

```diff
@@ -67,7 +67,7 @@ impl ExpressionAttributes {
 pub struct FunctionAttributes {
     pub name: String,
     pub param_types: Vec<FixedSize>,
-    pub return_type: Option<FixedSize>,
+    pub return_type: FixedSize,
```

Comment:
I like how this has streamlined a bunch of code. There's simply no `None` case anymore to handle..

Reply:
yeah, definitely nice not having to guard against these